dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -8,16 +8,21 @@
 DEFAULT_DATA_DICTIONARY_TEMPLATE_PATH = (
     RESOURCES_PATH / "data_dictionary_template.jinja"
 )
+DEFAULT_DATA_DICTIONARY_STYLESHEET_PATH = RESOURCES_PATH / "data_dictionary.css"
 
 
-def generate_pdf_from_html(output_html_path: Path, output_pdf_path: Path) -> Path:
+def generate_pdf_from_html(
+    output_html_path: Path,
```

Comment:
change to `input_html_path`?

Reply:
changed all of em to be shorter and consistent with `write_oti_xlsx`